tests/system.rs

```diff
@@ -249,8 +249,10 @@ fn handle_result(result: HashMap<String, String>,
 fn get_target(file_name: &str, target: Option<&str>, write_mode: WriteMode) -> String {
     let file_path = Path::new(file_name);
     let (source_path_prefix, target_path_prefix) = match write_mode {
-        WriteMode::Coverage => (Path::new("tests/coverage-source/"),
-                                "tests/coverage-target/"),
```

Comment:
I wonder if we should make an exception to the multi-line rule for literals some how? E.g., a multi-line if or match looks better in a block, but I feel like here the original is better. Just a thought for later improvement.